docs/platform/developing/creating.md

```diff
@@ -2,6 +2,17 @@
 
 This topic provides information on how to create a Tizen image.
 
+## MIC Image Creator
+
+MIC is an image creator that is used to create images for Tizen. 
+
+The tool offers three major functions as follows:
+- create an image in a specific format
+- chroot into an image
+- convert an image to another format
+
+For more information on the tool and its functions, see [MIC Image Creator](../../../reference/mic/mic-overview.md).
+
 Before creating an image, study the following instructions:
```

Comment:
This "Before creating an image, ..." line is kind of a prerequisite section in this page. 
So, how about moving the "MIC Image Creator" section after to this list like below image? 
Would you please let me know your opinion?

![image](https://user-images.githubusercontent.com/9958374/92433739-e8409d00-f1d8-11ea-90be-1e8c779591f6.png)

Reply:
@mijin85cho, your suggestion is good.

Reply:
done :)